From ce487b694e4b019aa881b58281954bd1d49c5ed3 Mon Sep 17 00:00:00 2001
From: Patrick Welche <prlw1@cam.ac.uk>
Date: Wed, 14 Apr 2010 18:09:00 +0000
Subject: Comment out CKeyboardHelper::Grab as not used after

b4cbd3b5 "Fixing up keyboard stuff" "TODO: Sort these methods out"
yet it needs XGrab from libX11. (#615573)
---
 
diff --git a/Src/Gtk2/KeyboardHelper.cpp b/Src/Gtk2/KeyboardHelper.cpp
index 50bd18d..937db58 100644
--- a/Src/Gtk2/KeyboardHelper.cpp
+++ b/Src/Gtk2/KeyboardHelper.cpp
@@ -167,6 +167,7 @@ int CKeyboardHelper::ConvertKeycode(int iCode) {
     return -1;
 }
 
+/* Not called - commented out because of X11 lib dependence
 void CKeyboardHelper::Grab(bool bGrab) {
   for(std::map<int,int>::iterator it(m_mTable.begin()); it != m_mTable.end(); ++it) {
     GdkKeymapKey *pKeys;
@@ -183,3 +184,4 @@ void CKeyboardHelper::Grab(bool bGrab) {
     }
   }
 }
+*/
diff --git a/Src/Gtk2/KeyboardHelper.h b/Src/Gtk2/KeyboardHelper.h
index c49e498..849088a 100644
--- a/Src/Gtk2/KeyboardHelper.h
+++ b/Src/Gtk2/KeyboardHelper.h
@@ -14,7 +14,7 @@ class CKeyboardHelper {
   void UnsetKey(int iCode);
 
   int ConvertKeycode(int iCode);
-  void Grab(bool bGrab);
+  // unused: void Grab(bool bGrab);
 
  private:
   void SetupDefaults();
--
cgit v0.9.0.2

